exercises/practice/decimal/tests/decimal.rs

```diff
@@ -36,7 +36,7 @@ fn test_ne() {
 fn test_gt() {
     for slice_2 in BIGS.windows(2) {
         assert!(decimal(slice_2[1]) > decimal(slice_2[0]));
-        assert!(!(decimal(slice_2[0]) > decimal(slice_2[1])));
+        assert!(decimal(slice_2[0]) < decimal(slice_2[1]));
```

Comment:
Well, at this point, `test_gt` and `test_lt` are testing the same thing, aren't they? May just need to delete this line. if `gt` should remain only about `>`.

Reply:
You're right; I'm not sure how one of those tests could possibly succeed without the other also succeeding. I guess I was trying to guard against a weird `Ord` implementation? That seems like not something to worry about.

Fixed in f3155346.

Reply:
I guess the reasoning there could be: Someone could potentially explicitly define `gt` to always return true, instead of relying on the implementation provided by the trait? If that sounds like something to actually test for, I would also approve something that ignores that particular lint for `test_gt` and `test_lt`

Reply:
Yeah, I'm not too concerned by that case. It's harder to do that than to implement `Ord` sensibly, so I'm willing to leave this to the mentors.